must-gather/collection-scripts/gather_node_info

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Command List
+commands_get=()
+
+# lvm commands
+commands_get+=("lvm lvs -a")
```

Comment:
we don't need to call lvm each time. For instance, just lvs, vgs etc will work. Why are we including lvm as well? lvm apparently opens the lvm shell.

Reply:
Lets check with the LVM dev if this could have any unwanted effects. Otherwise I think this is fine.

Reply:
I will check with him along with the other things, but as far as I have noticed running only "lvm" opens a lvm shell otherwise not. But yeah, all the commands work without prefixing lvm so I can remove that.